google-http-client/src/main/java/com/google/api/client/http/GenericUrl.java

```diff
@@ -99,24 +102,58 @@ public GenericUrl() {}
    * @throws IllegalArgumentException if URL has a syntax error
    */
   public GenericUrl(String encodedUrl) {
-    this(parseURL(encodedUrl));
+    this(encodedUrl, false);
+  }
+
+  /**
+   * Constructs from an encoded URL.
+   *
+   * <p>Any known query parameters with pre-defined fields as data keys will be parsed based on
```

Comment:
will be --> are (tech writing lives in the eternal present)

Reply:
I will fix this, however this doesn't seem to be the case in the existing codebase.

Reply:
In fact this is copied and pasted from the other constructor.